composer.json

```diff
@@ -34,7 +33,7 @@
         "laminas/laminas-session": "^2.8",
         "laminas/laminas-uri": "^2.5.2",
         "laminas/laminas-validator": "^2.10.1",
-        "phpunit/phpunit": "^9.3",
+        "phpunit/phpunit": "^9.3 <=9.5.9",
```

Comment:
It looks like we are closing our eyes here and ignoring the real problem. Is this the problem:

> PHPUnit no longer converts PHP deprecations to exceptions by default

https://github.com/sebastianbergmann/phpunit/blob/master/ChangeLog-9.5.md#9510---2021-09-25

Reply:
It appears I have a tendency to do this 🙈